tests/Executor/ExecutorTest.php

```diff
@@ -54,7 +54,7 @@ public function testExecutesArbitraryCode(): void
             });
         };
 
-        $data = [
+        $data     = [
```

Comment:
wasn't it compliant with CS already before?

Reply:
I tried reverting, seems like the newest version changes this. This is one of the reasons why I do not like alignment, my preference would be to left-align everything always.

Reply:
After some time, I agree. There's even no proper IDE support for this kind of alignment. It some cases it's really handy but overall not worth I guess.

Reply:
As far as I can tell, PHP_CodeSniffer has no option to force left-alignment. Wading through their docs is a total pain in the neck though, perhaps we should go with php-cs-fixer.